src/containers/Home.js

```diff
@@ -202,6 +204,15 @@ class Home extends Component {
                           </Link>
                         )}
                       </TableCell>
+                    ) : col.key === 'files_viewer' ? (
+                      <TableCell key={col.key}>
+                        <IconButton
+                          component={RouterLink}
+                          aria-label='files_viewer'
+                          to={`/files/${row.subject_id}`}>
```

Comment:
Can you hide or disabled this component / linking, if `subject_id` is null or undefined, pls. There are sample without Subject ID...

Reply:
Resolved in commit `bd8dc4a`